staging/src/k8s.io/apiserver/pkg/util/feature/feature_gate_test.go

```diff
@@ -124,7 +162,7 @@ func TestFeatureGateFlag(t *testing.T) {
 			testAlphaGate: {Default: false, PreRelease: Alpha},
 			testBetaGate:  {Default: false, PreRelease: Beta},
 		})
-		f.AddFlag(fs)
+		f.AddFlag(fs, "")
```

Comment:
can we also test the new functionality added by the second arg?

Reply:
I think this is being tested with the new KnownFeatures test now so i dont think its required to modify the current test.